jobs/mprsyncup

```diff
@@ -179,6 +196,13 @@ if [ "${PORTS_CHANGED}" -eq 1 ]; then
                 | expand -t 40,48,56,64,72,80 &
         done
         wait
+
+        # generate json for each platform-specific index
+        for PLATFORM in $PLATFORMS; do
+            INDEX="PortIndex_darwin_${PLATFORM}"
+            ${TCLSH} "${PORTINDEX2JSON}" "${INDEX}"/PortIndex > "${INDEX}"/PortIndex.json
```

Comment:
portindex2json should be run asynchronously, by using `&` at the end of the command, just like portindex is being run asynchronously above, so that several can run in parallel and make better use of the several CPU cores. Then the `wait` below will wait for those commands to finish before proceeding.

Reply:
The suggestion is ok, I just wanted to say that portindex2json is relatively fast compared to portindex, so it's not as critical as for generating portindex itself.